python/GangaPanda/PACKAGE.py

```diff
@@ -8,7 +8,7 @@
 """
 
 _external_packages = {
-    'panda-client' : { 'version' : '0.5.69',
+    'panda-client' : { 'version' : '0.5.80',
                     'syspath':['lib/python2.6/site-packages'],
                     'CONFIGEXTRACTOR_PATH':'etc/panda/share',
                     'PANDA_SYS':'.',
```

Comment:
I guess this is not the PR for it but it seems strange you do the exact same thing in two different files. Is there a reason?